pkg/clockcache/cache_test.go

```diff
@@ -0,0 +1,173 @@
+package clockcache
```

Comment:
This would be a great use case for some testing benchmarking functions too

Reply:
hmm, I have some benchmarks I copied of `bigcache` that I've been using, but I don't love them. It's a bit difficult to get benchmarks that are indicative of real-world contention, which is what should really matter for performance. I'll see what I can do...